modules/concept-docs/pages/durability-replication-failure-considerations.adoc

```diff
@@ -1,7 +1,7 @@
 = Durability & Failure
 :description: Data durability refers to the fault tolerance and persistence of data in the face of software or hardware failure.
 :page-topic-type: concept
-// :page-aliases: ROOT:failure-considerations,ROOT:durability,ROOT:enhanced-durability
+:page-aliases: ROOT:failure-considerations,ROOT:durability,ROOT:enhanced-durability
```

Comment:
just checking that you meant to uncomment these?
(As we discussed aliases for some other cases, and I see I didn't uncomment these in Node.js SDK, so checking if I *should*...)

Reply:
They were commented out initially, I believe, as there was nothing useful in the page - and possibly then overlooked.
I may be wrong - they could be duplicated in one of the howtos - but if Antora local test build finds no problem, then I'm happy to see them come back

Reply:
Thought we would need to un-comment these now, since the pages exist in the old 2.6 docs. 

- failure-considerations (https://docs-staging.couchbase.com/php-sdk/2.6/failure-considerations.html)
- durability (https://docs-staging.couchbase.com/php-sdk/2.6/durability.html)

Seems like `enhanced-durability` is on the Durability page though, so maybe I should remove `ROOT:enhanced-durability`? 
https://docs-staging.couchbase.com/php-sdk/2.6/durability.html#enhanced-durability

Reply:
My thinking was that 2.6 pages should inform that theres a new version of the page in 3.2. Antora builds fine also.